bookchain-miner/src/http/dtos/dtos.hpp

```diff
@@ -9,27 +8,14 @@ namespace bookchain::http {
 /* Begin DTO code-generation */
 #include OATPP_CODEGEN_BEGIN(DTO)
 
-class BasicMessageDto : public oatpp::data::mapping::type::Object {
-    DTO_INIT(BasicMessageDto, Object)
+class HelloMessageDto : public oatpp::data::mapping::type::Object {
+    DTO_INIT(HelloMessageDto, Object)
 
-    DTO_FIELD(Int32, statusCode);
-    DTO_FIELD(String, message);
-};
-
-class BlockDto : public oatpp::data::mapping::type::Object {
```

Comment:
Do we even need this file now that this is moved to blockdtos.hpp?

Reply:
I was keeping it for the general dtos, I have a feeling we will have things to put here. Also good for testing for now.